rueidis/rueidis_test.go

```diff
@@ -0,0 +1,217 @@
+package rueidis
+
+import (
+	"crypto/tls"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Should we use require or assert

Reply:
It will make the test end faster during failure, so sure. I can fix it

Reply:
Done